CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to pytket
+
+Pull requests are welcome. To make a PR, first fork the repo, make your proposed
+changes on the `main` branch, and open a PR from your fork. If it passes
+the checks and is accepted after review, it will be merged in.
+
+## pre-commit
+
+We use [pre-commit](https://pre-commit.com/#automatically-enabling-pre-commit-on-repositories) to catch formatting and other issues before committing to the repo. If you have installed the required packages with `pip install -r requirements.txt`, then you should already have `pre-commit` available.
+
+To install the hooks, simply run `pre-commit install`. After that, the checks will be run automatically whenever you do a `git commit`.
```

Comment:
Is there something that needs to be done if someone don't want to use that?

Reply:
Mmmm, they'd need to install `black` themselves and run it. Or just wait for the ci to catch any issues.

Reply:
I mean if it is still possible to commit to the git repo locally without getting any error messages if pre-commit is not installed?

Reply:
ah, sure, that'd still work as usual :)